learn/user-guide/ballerina-packages/creating-your-first-ballerina-package.md

```diff
@@ -18,15 +18,15 @@ The single source file program is a quick and easy way to get started with Balle
 To create a new Ballerina package, we use the bal new command:
 
 ```bash
-$ bal new hello_world
+$ bal new helloworld
 ```
 
 This will create a new Ballerina package with a main function. If we are creating a service instead of a main function, we pass the `-t service` option.
 Let’s check what the `bal new` command generated:
 ```bash
-$ cd hello_world
+$ cd helloworld
 $ tree .
-helloworld
+.
```

Comment:
The rest of the output also should align with the dot

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/2349/commits/59ad117c5c47aacfb2072bd9472a1c59852a62a3